cgmanifest.json

```diff
@@ -25004,8 +25004,8 @@
         "type": "other",
         "other": {
           "name": "quota",
-          "version": "4.06",
-          "downloadUrl": "http://downloads.sourceforge.net/linuxquota/quota-4.06.tar.gz"
+          "version": "4.09",
+          "downloadUrl": "http://downloads.sourceforge.net/linuxquota/quota-4.09.tar.gz"
```

Comment:
Is there a no `https` download link ?

Reply:
You could use the following: `https://downloads.sourceforge.net/project/linuxquota/quota-tools/4.09/quota-4.09.tar.gz`

Reply:
Good point!! URL updated to https.